src/database/queries/baseFields/insertOne.sql

```diff
@@ -10,10 +10,4 @@ VALUES (
   :shortCode,
   :dataType
 )
```

Comment:
I'm not necessarily advocating the following, but why wouldn't we have companian json_to_base_field functions for inserts as well?

Reply:
What value would that companion provide?  One benefit of this approach is we don't have to duplicate our casting of table-structure-to-json in more than one place, I think we'd lose that with a companion function, but maybe I'm misunderstanding what you mean!

Reply:
@slifty At a high level (without fully exploring or understanding the need for the json functions) having some consistency and symmetry between the readers and writers, as it were. If there is no need for it there is no need.